WooCommerce/Classes/ViewRelated/Blaze/BlazeCampaignList/BlazeCampaignIntroView.swift

```diff
@@ -0,0 +1,120 @@
+import SwiftUI
+
+/// View to display the introduction to the Blaze campaign
+///
+struct BlazeCampaignIntroView: View {
+    /// Scale of the view based on accessibility changes
+    @ScaledMetric private var scale: CGFloat = 1.0
+
+    var body: some View {
+        NavigationView {
+            ScrollView {
+                VStack(alignment: .leading) {
+                    Image(uiImage: .blaze)
+                        .frame(width: 100, height: 100)
+                        .clipShape(Circle())
+                        .background(
+                            Circle()
+                                .fill(Color(.withColorStudio(.orange, shade: .shade40)).opacity(0.08))
+                        )
+                        .padding(.bottom, Layout.elementVerticalSpacing)
+
+
+                    Text(Localization.title)
+                        .largeTitleStyle()
+                        .padding(.bottom, Layout.elementVerticalSpacing)
+
+                    BulletPointView(text: Localization.descriptionPoint1)
+                    BulletPointView(text: Localization.descriptionPoint2)
+                    BulletPointView(text: Localization.descriptionPoint3)
+                    BulletPointView(text: Localization.descriptionPoint4)
+                }
+                .padding(Layout.contentPadding)
+            }
+            .safeAreaInset(edge: .bottom) {
+                VStack {
+                    Divider()
+                        .frame(height: Layout.dividerHeight)
+                        .foregroundColor(Color(.separator))
+                    Button(Localization.startBlazeCampaign) {
+                        // todo
+                    }
+                    .buttonStyle(PrimaryButtonStyle())
+                    .padding(16)
+                }
+            }
+            .navigationBarTitleDisplayMode(.inline)
+            .toolbar {
+                ToolbarItem(placement: .cancellationAction) {
+                    Button(Localization.cancel) {
+                        // todo
+                    }
+                }
+            }
+        }
+    }
+}
+
+/// View to display a single text with a bullet point preceding it.
+///
+private struct BulletPointView: View {
+    let text: String
+
+    var body: some View {
+        HStack(alignment: .top, spacing: 4) {
+            Image(systemName: "circle.fill")
+                .font(.system(size: 4))
```

Comment:
We should avoid using a fixed font because it cannot scale. Please add `@ScaledMetric private var scale: CGFloat = 1.0` to this file and multiply the numbers with the scale.

<img width="563" alt="image" src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/9202bcfc-e9dd-4d56-8437-72cfd50a2bbd">

Reply:
Handled in https://github.com/woocommerce/woocommerce-ios/pull/10946/commits/a06cf6addb4d56e0ed30a550fd9e021d42131c21